### PR TITLE
BRS:224 

### PR DESCRIPTION
**Pull request by Christopher-walsh22**
### Jira Ticket:
BRS-224

### Jira Ticket URL:
https://github.com/orgs/bcgov/projects/49/views/1?pane=issue&itemId=36566634

### Description:
Added switch statement to handle converting numbers to month text.

### CODE REVIEW

File: lambda/export-variance/invokable/index.js [hunk 1]
```diff
@@ -282,4 +282,50 @@ async function uploadToS3(csvData) {
...
+    case 11:
+      monthName = 'November';
+      break;
+    case 12:
+      monthName = 'December';
+      break;
+    default:
+      monthName = `Invalid month: ${monthNumber}`;
+  }
+  return monthName
```
Comment (danieltruong) [inline lambda/export-variance/invokable/index.js:330]: Could you use an enum for this?

**Review by danieltruong — APPROVED** [review]

File: lambda/export-variance/invokable/index.js (hunk 1 above)
Comment (marklise) [reply lambda/export-variance/invokable/index.js:330]:
```
function getMonthName(monthNumber) {
  const months = [
    'January', 'February', 'March', 'April', 'May', 'June',
    'July', 'August', 'September', 'October', 'November', 'December'
  ];

  if (monthNumber >= 1 && monthNumber <= 12) {
    return months[monthNumber - 1];
  } else {
    return 'Invalid month number';
  }
}

// Example usage:
const monthNumber = 5; // May
const monthName = getMonthName(monthNumber);
console.log(monthName); // Output: "May"
```

Comment (marklise) [reply lambda/export-variance/invokable/index.js:330]: Alternatively:

```
function getMonthName(monthNumber) {
  if (monthNumber >= 1 && monthNumber <= 12) {
    const date = new Date(2000, monthNumber - 1, 1); // Year 2000 is an arbitrary year.
    return new Intl.DateTimeFormat('en-US', { month: 'long' }).format(date);
  } else {
    return 'Invalid month number';
  }
}

// Example usage:
const monthNumber = 5; // May
const monthName = getMonthName(monthNumber);
console.log(monthName); // Output: "May"

```

Comment (Christopher-walsh22) [reply lambda/export-variance/invokable/index.js]: Updated using your first example. Tested locally, requesting re-review!

**Review by cameronpettit — APPROVED** [review]

**Pull request merged**